docs/section22/Rule22-30.md

```diff
@@ -0,0 +1,58 @@
+
+# CHW&CW - Rule 22-30  
+
+**Schema Version:** 0.0.10 **Mandatory Rule:** True
+**Rule ID:** 22-30  
+**Rule Description:** For chilled-water systems served by chiller(s) and does not serve baseline System-11, condenser-water pump power shall be 19 W/gpm.  
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 22 CHW&CW Loop  
+**90.1 Section Reference:** Section G3.1.3.11 Heat Rejection (Systems 7, 8, 11, 12, and 13)  
+**Data Lookup:** None  
+**Evaluation Context:** Building  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with at least one air-side system that is Type-7, 8, 12, 13, 7b, 8b, 12b, 13b.
+2. B-RMR is not modeled with any system that is Type-11.1, 11.2 or 11b.
+3. Primary and secondary CHW loops are modeled correctly in B-RMR.
```

Comment:
This RDS is about condenser loop pump power, so why do we need to check primarly/secondary CHW loop?

Reply:
Primary/secondary loop check is now removed.